web/modules/custom/joinup_notification/joinup_notification.module

```diff
@@ -145,10 +145,12 @@ function joinup_notification_mail_alter(&$message) {
   if ($message['module'] === 'message_notify') {
     $message['headers']['Content-Type'] = SWIFTMAILER_FORMAT_HTML;
     $user = user_load_by_mail($message['to']);
-    $first_name = !empty($user->get('field_user_first_name')->value) ? $user->get('field_user_first_name')->value : 'Joinup user';
-    $prefix = t('<p>Dear @first_name,</p>', ['@first_name' => $first_name]);
-    $suffix = t('<p>Kind regards,</p><p>The Joinup Support Team</p>');
-    $message['body'] = array_merge([$prefix], $message['body'], [$suffix]);
+    if ($user) {
+      $first_name = !empty($user->get('field_user_first_name')->value) ? $user->get('field_user_first_name')->value : 'Joinup user';
+      $prefix = t('<p>Dear @first_name,</p>', ['@first_name' => $first_name]);
+      $suffix = t('<p>Kind regards,</p><p>The Joinup Support Team</p>');
+      $message['body'] = array_merge([$prefix], $message['body'], [$suffix]);
```

Comment:
This code is only clear to people that have knowledge about the ticket ISAICP-3981.

To me it seems clear that if there is no `$user` associated with the `to:` field that we cannot greet the user by their first name, but it is not clear why this should affect the signature as well.

Can we make this more clear? Either with documentation, or by hardcoding the `contact_form_submission` template, or (ideally) by making the signature an optional feature that defaults to being present?